test/test_examples_2d.jl

```diff
@@ -120,10 +120,21 @@ const EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples", "
   end
 
   @testset "elixir_euler_khi_shockcapturing_amr.jl with tend = 0.2" begin
-  test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_euler_khi_shockcapturing_amr.jl"),
-    l2   = [0.0019809356303815313, 0.006538462481807526, 0.004737804472678921, 0.0050181776990539505],
-    linf = [0.016342197215556853, 0.03993613023503173, 0.015293069044755532, 0.024177402362647094],
-    tspan = (0.0, 0.2))
+  if Threads.nthreads() == 1
```

Comment:
Is this check because `compute_coefficients` is multi-threaded?

Reply:
And do you see any solution for this? I don't, since if we use multi-threading, the only reproducible order could be achieved by using a separate PRNG for each element, which seems a tad much...

Reply:
> Is this check because `compute_coefficients` is multi-threaded?

Yes. The tests fails on my machine using multiple threads.



> And do you see any solution for this? I don't, since if we use multi-threading, the only reproducible order could be achieved by using a separate PRNG for each element, which seems a tad much...

No, I don't see any other solution right now. The only solution would be to reduce the test to something that does not depend on the concrete stream of random numbers but only on their general distribution - such that it's safe enough in general...

Reply:
OK, then this is the best option for now.

Reply:
On the other hand, it's a very specific application setup that isn't used to test anything not covered by other tests. Hence, I think it's okay to not be very strict here.